lib/boss/boss_store.rb

```diff
@@ -17,59 +17,36 @@ class BOSSStorage < Ruote::FsStorage
     attr_reader :number
 
     def initialize(dir, options={})
+      if File.stat(dir).uid != Process.euid
```

Comment:
ideally this would do a recursive check...

Reply:
Hopefully it will not so easily end up under inconsistent ownership after this, so scanning the whole tree shouldn't be needed. And hopefully we'll have soon time to investigate other storage options :)